core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/workspace/runners/BPMBillingProjectDeleteRunner.scala

```diff
@@ -63,24 +62,18 @@ class BPMBillingProjectDeleteRunner(
           .updateCreationStatus(projectName, DeletionFailed, Some(errorMsg))
           .map(_ => Complete)
     }
+
     getUserCtx(userEmail).transformWith {
       case Success(userCtx) =>
-        job.jobType match {
-          case AzureBillingProjectDelete =>
-            billingRepository.getLandingZoneId(projectName).flatMap {
-              case Some(landingZoneId) =>
-                handleLandingZoneDeletion(job.jobControlId, projectName, UUID.fromString(landingZoneId), userCtx)
-              case None =>
-                val msg = s"No landing zone id set on Azure billing project"
-                logger.error(msg)
-                billingRepository.updateCreationStatus(projectName, DeletionFailed, Some(msg)).map(_ => Complete)
-            }
-          case _ => billingLifecycle.finalizeDelete(projectName, userCtx).map(_ => Complete)
+        billingRepository.getLandingZoneId(projectName).flatMap {
+          case Some(landingZoneId) =>
+            handleLandingZoneDeletion(job.jobControlId, projectName, UUID.fromString(landingZoneId), userCtx)
+          case None => billingLifecycle.finalizeDelete(projectName, userCtx).map(_ => Complete)
         }
       case Failure(t) =>
         val msg = s"Unable to complete billing project deletion: unable to retrieve request context for $userEmail"
         logger.error(s"${job.jobType} job ${job.jobControlId} for billing project: $projectName failed: $msg", t)
-        billingRepository.updateCreationStatus(projectName, DeletionFailed, Some(msg)).map(_ => Incomplete)
+        billingRepository.updateCreationStatus(projectName, Deleting, Some(msg)).map(_ => Incomplete)
```

Comment:
There's a few other cases where we're setting the status as `DeletionFailed` with an `Incomplete` status. They all look like similar cases where we expect the failure to be transient and we want to retry. Can we keep those statuses as `Deleting` as well? I think `DeletionFailed` will be a more helpful status for us if we can treat it as a fatal error that requires some manual intervention

Reply:
@blakery @marctalbott did this get followed up on?